modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -235,8 +235,39 @@ inline GRunArgsP& operator += (GRunArgsP &lhs, const GRunArgsP &rhs)
 
 namespace gapi
 {
-    GAPI_EXPORTS cv::GRunArgsP bind(cv::GRunArgs &results);
+    /**
+     * \addtogroup gapi_serialization
+     * @{
+     *
```

Comment:
It is better to avoid indentation in namespaces.

Reply:
Right, fixed